vars/checkValidateCommit.groovy

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env groovy
+
```

Comment:
checkValidateCommit.groovy

improve script name to 'validateCommits.groovy'

checkValidateCommits is a bad name (don't use two consecutive verbs)